### PR TITLE
Initial onboarding script for VSTS

### PR DESCRIPTION
**Pull request by mmitche**
Onboards a GitHub repo into VSTS public/internal.
Currently doesn't create build definitions

### CODE REVIEW

File: scripts/vsts-onboard.ps1 [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+.SYNOPSIS
+Performs a set of actions to enable a repo in VSTS.
+
+.DESCRIPTION
+- Makes an internal repo on dotnet.visualstudio.com for the github repo
+- Adds a Maestro webhook
+- Adds a folder structure on internal and public vsts for build definitions
+
+.PARAMETER GitHubRepoName
+Repository name of the github repo to onboard, in org/repo form.=
```
Comment (JohnTortugo) [inline scripts/vsts-onboard.ps1]: NIT. "="

File: scripts/vsts-onboard.ps1 [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    $githubHookCreate = Invoke-WebRequest -Method Post "https://api.github.com/repos/$GitHubRepoName/hooks$githubTokenSuffix" -Body "{`"name`": `"web`", `"active`": true, `"events`": [`"push`"], `"config`": { `"url`": `"$maestroHookUrl`", `"content_type`": `"json`", `"secret`":`"$maestroHookSecret`" } }"
+    if ($githubHookCreate.StatusCode -ne 201) {
+        Write-Error "Failed to create Maestro webhook for $GithubRepoName"
+        exit
+    } else {
+        Write-Output "Created Maestro webhook for $GitHubRepoName"
+    }
+}
+
+# Create folders in the public project and internal project
```
Comment (JohnTortugo) [inline scripts/vsts-onboard.ps1]: I think this comment isn't adding much here :-)

File: scripts/vsts-onboard.ps1 [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+Write-Output "Creating folders in public and internal projects"
```
Comment (JohnTortugo) [inline scripts/vsts-onboard.ps1]: Maybe omit this line if DryRun is set?

Comment (mmitche) [reply scripts/vsts-onboard.ps1]: It was intentional.  DryRun doesn't perform any changes.  However, it should still do the existence checks, so i've moved this check inside Create-VSTS-Folder

File: scripts/vsts-onboard.ps1 (hunk 1 above)
Comment (mmitche) [reply scripts/vsts-onboard.ps1]: Fixed

File: scripts/vsts-onboard.ps1 (hunk 2 above)
Comment (mmitche) [reply scripts/vsts-onboard.ps1]: Removed

File: scripts/vsts-onboard.ps1 [hunk 4]
```diff
@@ -0,0 +1,171 @@
+<#
```
Comment (weshaggard) [inline scripts/vsts-onboard.ps1]: Do these scripts need to be in the root directory? Can we put them under `eng`?

**Pull request merged**